lib/hammer_cli_katello/host_extensions.rb

```diff
@@ -90,6 +89,15 @@ module HostExtensions
           field :id, _('Id')
           field :name, _('Name')
         end
+
+        label _('System Purpose') do
```

Comment:
Perhaps we should nest this under `Subscription Information`?

Reply:
Yes. That makes sense. Thanks. :+1: